nodeclient/core_models.go

```diff
@@ -93,113 +108,82 @@ type (
-		MilestoneIndexSpent iotago.SlotIndex `json:"milestoneIndexSpent,omitempty"`
-		// The milestone timestamp this output was spent.
-		MilestoneTimestampSpent uint32 `json:"milestoneTimestampSpent,omitempty"`
-		// The transaction this output was spent with.
+		// IsSpent indicates whether the output is spent or not.
+		IsSpent bool `json:"isSpent"`
+		// CommitmentIDSpent is the commitment ID that includes the spent output.
+		CommitmentIDSpent string `json:"commitmentIdSpent,omitempty"`
+		// TransactionIDSpent is the transaction ID that spends the output.
 		TransactionIDSpent string `json:"transactionIdSpent,omitempty"`
-		// The milestone index at which this output was booked into the ledger.
-		MilestoneIndexBooked iotago.SlotIndex `json:"milestoneIndexBooked"`
-		// The milestone timestamp this output was booked in the ledger.
-		MilestoneTimestampBooked uint32 `json:"milestoneTimestampBooked"`
-		// The ledger index at which this output was available at.
-		LedgerIndex iotago.SlotIndex `json:"ledgerIndex"`
-	}
-
-	// OutputResponse defines the response of a GET outputs REST API call.
-	OutputResponse struct {
-		Metadata *OutputMetadataResponse `json:"metadata"`
-		// The output in its serialized form.
-		RawOutput *json.RawMessage `json:"output"`
-	}
-
-	// TreasuryResponse defines the response of a GET treasury REST API call.
-	TreasuryResponse struct {
-		MilestoneID string `json:"milestoneId"`
-		Amount      string `json:"amount"`
-	}
-
-	// ReceiptsResponse defines the response for receipts GET related REST API calls.
-	ReceiptsResponse struct {
-		Receipts []*ReceiptTuple `json:"receipts"`
+		// IncludedCommitmentID is the commitment ID that includes the output.
+		IncludedCommitmentID string `json:"includedCommitmentId,omitempty"`
+		// LatestCommitmentID is the latest commitment ID of a node.
+		LatestCommitmentID string `json:"latestCommitmentId"`
 	}
 
-	// ReceiptTuple represents a receipt and the milestone index in which it was contained.
-	ReceiptTuple struct {
-		Receipt        *json.RawMessage `json:"receipt"`
-		MilestoneIndex iotago.SlotIndex `json:"milestoneIndex"`
+	// CommitmentUTXOChangesResponse defines the response of a GET milestone UTXO changes REST API call.
+	CommitmentInfoResponse struct {
+		// The index of the requested commitment.
+		Index iotago.SlotIndex `json:"index"`
+		// The commitment ID of previous commitment.
+		PrevID string `json:"prevId"`
+		// The roots ID of merkle trees within the requested commitment.
+		RootsID string `json:"rootsId"`
+		// The cumulative weight of the requested slot.
+		CumulativeWeight uint64 `json:"cumulativeWeight"`
+		// TODO: decide what else to add here.
 	}
 
-	// CommitmentUTXOChangesResponse defines the response of a GET milestone UTXO changes REST API call.
-	CommitmentUTXOChangesResponse struct {
-		// The index of the milestone.
+	// SlotUTXOResponse defines the response for UTXO slot REST API call.
+	SlotUTXOResponse struct {
```

Comment:
In Swagger this is called `UTXOChangesResponse`. I think we should keep the naming consistent between code and specs.